### PR TITLE
Feat/create app carousel

### PR DESCRIPTION
**Pull request by NNaumenko83**
Hi, I have created the AppCarousel component. 
Additionally, I have added CarouselChefs and CarouselDishes with mock arrays to check how it works. 
I used **react-slick** for the carousel. 
To style some elements, I need to add a CSS file.
Unfortunately, our DishCard and ChefCard components have fixed dimensions and do not depend on screen size for now.

### CODE REVIEW

File: src/components/CarouselDisches/CarouselDisches.jsx
Comment (kyzym) [inline src/components/CarouselDisches/CarouselDisches.jsx:1]: A little misprint - Disches -  **dishes**

**Review by kyzym — APPROVED** [review]
Looks great.

File: src/components/CarouselDisches/CarouselDisches.jsx
Comment (NNaumenko83) [reply src/components/CarouselDisches/CarouselDisches.jsx:1]: Thank you. 
Changed.

**Review by vladochka2812 — APPROVED** [review]

**Review by NikolayVerstak — APPROVED** [review]
Nice)

**Pull request merged**